### PR TITLE
[EN/ID] Add "Hard Rock" and "Half Time" mod articles, fix mistake in the original "Hard Rock" article

### PR DESCRIPTION
**Pull request by wowcake**
Added the ID translation of Hard Rock and Half Time,

And correct a slight mistake on the EN version of Hard Rock.

### CODE REVIEW

File: wiki/Game_modifier/Hard_Rock/id.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+*Perhatian: Menggunakan mod ini pada mode osu!mania akan menyebabkan sebuah unranked play.*
+
+Pada [osu!mania](/wiki/Game_mode/osu!mania), hanya pengaturan kesulitan penilaian (OD) dan nilai AR yang ditingkatkan. Pengaturan kesulitan lainnya tetap tidak berubah. Posisi peletakkan not-not tidak terpengaruhi.
+
+## Trivia
+
+- Mod Hard Rock dinamai dari kesulitan tersusah pada permainan [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS "Wikipedia"), "[Elite Beat Agents](/wiki/Glossary#eba)"
+- Mod Hard Rock akan mematikan mod [Easy](/wiki/Game_modifier/Easy). Dan sebaliknya.
+
+[osu!]: /wiki/shared/mode/osu.png "osu!standard"
```
Comment (mousesalt) [inline wiki/Game_modifier/Hard_Rock/id.md:67]:
```suggestion
[osu!]: /wiki/shared/mode/osu.png "osu!"
```

File: wiki/Game_modifier/Hard_Rock/id.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan kebanyakan aspek (jika tidak semua) kesulitan dari sebuah [beatmap](/wiki/Beatmap). Mod ini melakukan hal itu dengan meningkatkan [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) sebesar 30%, dan pengaturan kesulitan lainnya sebesar 40%.
+
+### osu!
+
+Pada [osu!](/wiki/Game_mode/osu!), (bersama dengan pengaturan kesulitan yang telah ditingkatkan) mod Hard Rock membalikkan posisi sumbu-X sebuah beatmap (ditunjukkan dibawah). Hal ini kadang-kadang bisa menyebabkan [pola](/wiki/Beatmap/Pattern) tertentu terasa tidak benar, yang bisa meningkatkan kesulitan.
+
+Sebagai tambahan, jika pemain telah menghafal atau sudah terbiasa dengan gerakan dan pola sebuah beatmap, beralih ke mod Hard Rock bisa menghilangkan penghafalan tersebut dan menyebabkan semacam sebuah kesulitan palsu.
+
+![Tampilan permainan HR](img/HR-comparison-standard.jpg "Perbandingan dari sebuah map tanpa mod terpasang (kiri) dengan map yang sama dengan mod Hard Rock dipasang (kanan) pada mode osu!")
```
Comment (mousesalt) [inline wiki/Game_modifier/Hard_Rock/id.md:38]:
```suggestion
![Tampilan permainan HR](img/HR-comparison-osu.jpg "Perbandingan dari sebuah map tanpa mod terpasang (kiri) dengan map yang sama menggunakan mod Hard Rock dipasang (kanan) pada mode osu!")
```

**Review by mousesalt — CHANGES_REQUESTED** [review]

File: wiki/Game_modifier/Half_Time/id.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  - ![][osu!mania]: 0.50x
+- Tombol Shortcut Default: `E`
+- Caption: `Mode Siput.`
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## Deskripsi
+
+*Catatan: Cara yang digunakan untuk memperlambat lagu tersebut bisa mengakibatkan lagunya kedengaran seperti robot atau agak keruh.*
+
+Mod **Half Time** adalah sebuah [game modifier](/wiki/Game_modifier) yang mengurangi kecepatan tempo (BPM) dari sebuah [beatmap](/wiki/Beatmap) ke 75% dari kecepatan awalnya, meingkatkan durasi dari lagu tersebut sebesar 33%, dan mengurangi sedikit jumlah [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty), [dan health (HP)](/wiki/Beatmapping/Health).
```
Comment (mousesalt) [inline wiki/Game_modifier/Half_Time/id.md:31]:
```suggestion
**Half Time** adalah sebuah [game modifier](/wiki/Game_modifier) yang berfungsi mengurangi kecepatan tempo (BPM) dari sebuah [beatmap](/wiki/Beatmap) menjadi 75% dari kecepatan awalnya, dengan meningkatkan durasi lagu sebesar 33%, dan mengurangi sedikit jumlah dari [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty), [dan health (HP)](/wiki/Beatmapping/Health).
```

File: wiki/Game_modifier/Half_Time/id.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+### osu!taiko
+
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), lagu dan tempo yang sudah diperlambat menyebabkan not-not menjadi sangat bertumpuk karena AR yang telah dikurangi. Tetapi, karena cara denden dikalkulasikan pada osu!taiko, mereka sekarang memerlukan lebih banyak ketukan untuk diselesaikan dan mengakibatkan mereka untuk memberikan **jumlah nilai yang lebih tinggi daripada tanpa menggunakan mod** karena ketukan denden tidak terpengaruhi oleh [pengali skor](/wiki/Game_modifier/Score_multiplier).
```
Comment (mousesalt) [inline wiki/Game_modifier/Half_Time/id.md:35]:
```suggestion
Pada [osu!taiko](/wiki/Game_mode/osu!taiko), lagu dan tempo yang sudah diperlambat menyebabkan not-not menjadi sangat bertumpuk karena jumlah dari approach rate (AR) telah dikurangi. Namun, dikarenakan kalkulasi denden pada osu!taiko memerlukan lebih banyak ketukan untuk diselesaikan, mengakibatkan pemberian **jumlah nilai menjadi lebih tinggi daripada tanpa menggunakan mod**, karena ketukan denden tidak dipengaruhi oleh [pengali skor](/wiki/Game_modifier/Score_multiplier).
```

Comment (mousesalt) [reply wiki/Game_modifier/Half_Time/id.md:35]: reword this, seems really strange to understand.

File: wiki/Game_modifier/Half_Time/id.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Karena itu, menggunakan mod Half Time bisa menyebabkan peningkatan dalam skor maksimum yang dapat dicapai di semua map dengan kombo yang sangat rendah dan jumlah denden panjang yang banyak. Efek ini bisa dilihat secara lebih besar ketika mod [Hard Rock](/wiki/Game_modifier/Hard_Rock) digunakan secara bersamaan.
+
+### osu!catch
+
+Pada [osu!catch](/wiki/Game_mode/osu!catch), tempo nya diturunkan oleh faktor yang sama seperti di mode-mode permainan lainnya. Tetapi mod ini *juga* mengurangi kecepatan pemain, jadi fruit biasa tanpa mod lainnya *bisa* berubah menjadi sebuah fruit hyperdash.
```
Comment (mousesalt) [inline wiki/Game_modifier/Half_Time/id.md:41]:
```suggestion
Pada [osu!catch](/wiki/Game_mode/osu!catch), lagu dan tempo juga direndahkan oleh faktor yang sama seperti di mode-mode permainan lainnya. Tetapi mod ini *juga* mengurangi kecepatan pemain, jadi fruit biasa tanpa mod lainnya *bisa* berubah menjadi sebuah fruit hyperdash.
```

File: wiki/Game_modifier/Hard_Rock/id.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  - ![][osu!] ![][osu!taiko]: 1.06x
+  - ![][osu!catch]: 1.12x
+  - ![][osu!mania]: 1.00x
+- Tombol Shortcut Default: `A`
+- Caption: `Semuanya menjadi lebih sulit...`
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## Deskripsi
+
+Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan kebanyakan aspek (jika tidak semua) kesulitan dari sebuah [beatmap](/wiki/Beatmap). Mod ini melakukan hal itu dengan meningkatkan [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) sebesar 30%, dan pengaturan kesulitan lainnya sebesar 40%.
```
Comment (mousesalt) [inline wiki/Game_modifier/Hard_Rock/id.md:30]:
```suggestion
Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan aspek-aspek (jika tidak semua) kesulitan dari sebuah [beatmap](/wiki/Beatmap). Mod ini meningkatkan [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) sebesar 30%, dan pengaturan kesulitan lainnya sebesar 40%.
```

Comment (mousesalt) [reply wiki/Game_modifier/Hard_Rock/id.md:30]: `most` and `aspects` double meaning

**Review by mousesalt — COMMENTED** [review]
rest is fine

File: wiki/Game_modifier/Half_Time/id.md (hunk 4 above)
Comment (wowcake) [reply wiki/Game_modifier/Half_Time/id.md:35]: kinda got confused there aswell

**Pull request merged**